docs/it-manual/infrastructure-requirements.md

```diff
@@ -6,6 +6,12 @@ CiviForm is a classic web-based client-server architecture which stores applican
 
 ### The Subsystems
 
+Basic architecture
+
+![Screenshot from 2022-05-17 12-37-22](https://user-images.githubusercontent.com/473671/168895920-7662b8cd-83b0-4437-ae26-961e37450e59.png)
```

Comment:
Looks great! I wonder if we should commit to using the `.gitbook/assets` directory for files rather than the GH upload. I don't think it's important either way.

Reply:
I took a couple minutes to figure out how to use the assets folder and didn't find a ready explanation in the gitbook docs. Suggestions?

Reply:
I need to write meta-docs, but it's where gitbook automatically puts files that are uploaded through its interface. However, you can just as well manually put files in that directory and link to them using relative links in markdown.

Reply:
Ah hmm, does that mean previewing the markdown in the github PR won't work?

Reply:
Preview in the GH markdown editor works the same, as long as the file is in the directory (though that may require one more step than using the built-in upload, and may be inaccessible to less technical people).

Reply:
Ah cool, I'm fine with either then. If you'd prefer using the assets dir I'll update the PR

Reply:
I think let's not worry about it, people should use whatever is easiest for them, and if we need to standardize for whatever reason going forward it is easy to refactor!

Reply:
sounds good